doc/versions.json

```
@@ -35,6 +36,15 @@
       "tfm": "1.5.0",
       "matter": ""
     },
+    "1.9.2-dev2": {
+      "ncs": "1.9.2-dev1",
+      "nrfxlib": "1.9.2-dev1",
```

I don't think this is updated for the dev tag releases
```suggestion
      "nrfxlib": "1.9.99",
```

---

Reverting, but would like confirmation from @rlubos .

---

Actually, I don't know. I don't see any entries for older dev tags, but v2.2.99-dev3 **has** an entry on main:
https://github.com/nrfconnect/sdk-nrf/blob/main/doc/versions.json#L47

The process guidelines are not too specific:
> Doc needs updated for release highlights and **version update**. 

@shantha-14 Can you confirm what to do here?